rust_src/src/casefiddle.rs

```diff
@@ -0,0 +1,26 @@
+//! Case conversion functions.
+use lisp::defsubr;
+use lisp::LispObject;
+use remacs_macros::lisp_fn;
+use remacs_sys::{casify_object, CaseAction};
+
+/// Convert argument to upper case and return that.
+/// The argument may be a character or string.  The result has the same type.
+/// The argument object is not altered--the value is a copy.  If argument
+/// is a character, characters which map to multiple code points when
+/// cased, e.g.  , are returned unchanged.
+/// See also `capitalize', `downcase' and `upcase-initials'.  */
```

Comment:
Also, the character ﬁ is missing from the comment.

Reply:
Another good catch. That is also in a01e26